### PR TITLE
Breaking/separate node and compiler

### PR DESCRIPTION
**Pull request by HristiyanG**

### CODE REVIEW

File: test/commands-tests/test.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ describe('AEproject Test', () => {
         fs.copySync(dockerPath, `${ nodeCWD }/docker`);
 
         executeOptions.cwd = nodeCWD
-        await execute(constants.cliCommands.NODE, [], executeOptions);
+        await execute(constants.cliCommands.ENV, [], executeOptions);
```
[the remark this answers is not in the document]
Comment (HristiyanG) [reply test/commands-tests/test.js]: we could not deploy the contract without the compiler running.

File: packages/aeproject-cli/aeproject-env/EnvService.js [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+            default:
+                spawn('docker-compose', ['-f', 'docker-compose.yml', '-f', 'docker-compose.compiler.yml', 'up', '-d']);
+                nodeService.save();
+        }
+    }
+
+    async stopAll () {
+
+        if (nodeService.getNodePath() && nodeService.getCompilerPath()) {
+            await this.stopNode();
```
[the remark this answers is not in the document]
Comment (HristiyanG) [reply packages/aeproject-cli/aeproject-env/EnvService.js]: Should be ok. Processes are run asynchronously

File: packages/aeproject-cli/aeproject-env/env/env.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+            // if not running, current env may be windows
+            // to reduce optional params we check is it running on windows env
+            if (!running) {
+                running = await this.areNodeAndCompilerRunning(dockerImage, compilerImage)
+                printError(`===== Compiler or Node is not running! ===== \n===== Please run the relevant command for your image! =====`)
+                return
+            }
+
+            if (!running) {
```
[the remark this answers is not in the document]
Comment (maptuhec) [reply packages/aeproject-cli/aeproject-env/env/env.js]: THis is still valid

File: packages/aeproject-cli/aeproject-env/env/node/node.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+    async run (option) {
+
+        let dockerImage = option.windows ? nodeConfiguration.dockerServiceNodeName : nodeConfiguration.dockerImage;
+        dockerImage = nodeConfiguration.dockerServiceNodeName;
+
+        try {
+            let running = await super.waitForContainer(dockerImage);
+
+            if (option.info) {
+                await super.printInfo(running)
```
[the remark this answers is not in the document]
Comment (maptuhec) [reply packages/aeproject-cli/aeproject-env/env/node/node.js]: this is also valid

**Pull request merged**